process.py

```diff
@@ -163,6 +165,10 @@ def playWithOMX(url, sub, width="", height="", new_log=False):
             "omxplayer -b -r -o both '" + url + "' " + resolution + " --vol " +
             str(volume) + " < /tmp/cmd"
         )
+    if "/tmp/" in url:
+        os.system(
+            "rm -rf " + url
```

Comment:
This line sort of scares me - is there not a safer way to do this? :slightly_smiling_face:

Reply:
You know, you're right. That wasn't a good idea. Sorry, this is my first time playing around with python, I'm still getting used to it. I'll look into it!